docs/accordproject-developers.md

```diff
@@ -3,4 +3,22 @@ id: accordproject-developers
 title: For Developers
 ---
 
-**TBD**
+## Why is the Accord Project relevant for developers?
+The Accord Project provides a universal format for smart legal contracts, and this format is embodied in a variety of open source projects that comprise the Accord Project technology stack. Input from developers are crucial for the Accord Project.
+
+Developers can contribute by translating legal text into corresponding computer code, creating Accord Project templates to be used by lawyers and businesses. In addition, developers can provide input on the development of the technology stack components: language, models, templating, and other tools.
```

Comment:
Not sure about the word `translating` (I wished I had something concrete to offer as alternative), but it should probably rewritten a bit.